packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -1,5 +1,5 @@
 <template>
```

Comment:
Remove the old DS used by the components in the demo.
I see classes on:

- `x-components/src/views/home/result.vue` -> Use here `x-picture-zoom`
- `x-components/src/x-modules/next-queries/components/next-query-preview.vue`
- `x-components/src/x-modules/queries-preview/components/query-preview.vue`
- `x-components/tests/unit/base-result-image.spec.ts`

Some of them are only in the component docs, but I think we should remove them from there as well to avoid confusion.

Reply:
@ajperezbau, @mnavarroespinosa I left the class `x-result__picture` as it comes from the `result.vue` component. Let me know if I'm missing something please!

Reply:
Yep, you are right, `x-result__picture` will not disappear til the `result` xds component is deprecated.